src/components/Auth/AuthComponent.js

```diff
@@ -1,4 +1,7 @@
 /* eslint-disable no-unused-vars */
+/* ignore Invalid property 'align' found on tag 'div' */
+/* eslint-disable react/no-unknown-property */
```

Comment:
@giranm I briefly looked at the CSS equivalent of this, decided it was better to ignore 😉

Reply:
😂  Allow it.